pkg/gui/controllers/helpers/mode_helper.go

```diff
@@ -46,6 +46,24 @@ type ModeStatus struct {
 	Reset       func() error
 }
 
+func (self *ModeHelper) getFilter() string {
```

Comment:
This should return two strings: one for the type of thing we're filtering by, and one for the value. That removes the ambiguity